<launch>
  <node pkg="orsens" type="orsens_node" name="orsens"  output="screen" respawn="true">
    <param name="capture_mode" value="depth_left"/>    
    <param name="color_width" value="640"/>
    <param name="depth_width" value="640"/> 
    <param name="compress_color" value="false"/> 
    <param name="compress_depth" value="false"/> 
    <param name="publish_depth" value="false"/> 
    <param name="publish_cloud" value="false"/> 
    <param name="data_path" value="$(find orsens)/data"/>
  </node> 
  <node pkg="image_view" type="image_view" name="color">
    <remap from="image" to="orsens/left/image_raw" />
    <param name="autosize" value="true" />
  </node> 
  <node pkg="image_view" type="image_view" name="depth">
    <remap from="image" to="orsens/disp" />
    <param name="autosize" value="true" />
  </node> 
 </launch>
